packages/ui/src/lib/NativeBase/Navbar.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  Image,
+  Box,
+  HamburgerIcon,
+  useMediaQuery,
+  HStack,
+  Button,
+  Link,
+  Square,
+} from "native-base";
+import PropTypes, { InferProps } from "prop-types";
+import React, { useReducer } from "react";
+
+const menuItemPropTypes = {
+  name: PropTypes.string.isRequired,
+  url: PropTypes.string.isRequired,
+};
+
+type MenuItemProps = InferProps<typeof menuItemPropTypes>;
+
+function MenuItem({ name, url }: MenuItemProps) {
+  return (
+    <Box alignItems="flex-start">
+      <Link
+        href={url}
+        px="16px"
+        py="10px"
+        fontSize="16px"
+        isUnderlined={false}
+        _text={{ color: "#FFFFFF" }}
+        _hover={{ _text: { color: "#00C2FF" } }}
```

Comment:
Let's utilise styled system from native base to remove component specific style for reusability and consistency.